### DISCUSSION TITLE
How to use Scalar Any or Map as mutation input type?

### DISCUSSION THREAD

**Discussion by twuyts** [99designs/gqlgen#1806] [Q&A]
### What happened?
I would like to have a mutation where the caller can pass a Map of strings, or even Any json object.
The problem is not defining this in gqlgen, the issue is how to call it?
In the playground, I try this:
```
mutation {
  mapExample(input:{"a":1})
}
```
but unfortunately, that gives me an error: 
```
{"errors":[{"message":"Expected Name, found String","locations":[{"line":2,"column":22}],"extensions":{"code":"GRAPHQL_PARSE_FAILED"}}],"data":null}
```
because clearly, it does not know how to validate the 'Map' scalar.

### What did you expect?
I would expect it to have just passed the Map to my resolver

### Minimal graphql.schema and models to reproduce
```
scalar Map
type Mutation {
  mapExample(input: Map!): String!
}
```
Resolver:
```
func (r *mutationResolver) MapExample(ctx context.Context, input map[string]interface{}) (string, error) {
	s := "the map looks like this: "
	for k, v := range input {
		s = fmt.Sprintf("%s %s-%v", s, k, v)
	}
	return s, nil
}
```
### versions
 - `gqlgen version`: v0.13.0 
 - `go version`: go version go1.16 linux/amd64
  - dep or go modules?
  ```
  module github.com/Klarrio/gqlgen-todos

go 1.16

require (
	github.com/99designs/gqlgen v0.13.0 // indirect
	github.com/vektah/gqlparser/v2 v2.1.0 // indirect
)
```

**Comment by duckbrain** [comment]
GRAPHQL_PARSE_FAILED is complaining about the query's syntax. Unlike JSON, The literal `{"a":1}` shouldn't have the quotes around "a".
